BankClient/UserInformation.cs

```diff
@@ -1,28 +1,28 @@
-﻿namespace BankClient;
+﻿using System;
+
+namespace BankClient;
 
 public class UserInformation
 {
     public UserInformation()
     {
-        Console.WriteLine("Firstname: ");
-        FirstName = Console.ReadLine();
-        Console.WriteLine("Lastname: ");
-        LastName = Console.ReadLine();
-        Console.WriteLine("Phonenumber: ");
-        Phonenumber = Console.ReadLine();
-        ID = Guid.NewGuid();
-        Console.WriteLine("The client was added");
+        
```

Comment:
You can add Id generation in this constructor

Reply:
Ok